docs/cookbook/custom-email.rst

```diff
@@ -0,0 +1,176 @@
+How to send a custom e-mail?
+============================
+
+.. note::
+
+    This cookbook is suitable for a clean :doc:`sylius-standard installation </book/installation>`.
+    For more general tips, while using :doc:`SyliusMailerBundle </bundles/SyliusMailerBundle/index>`
+    go to `Sending configurable e-mails in Symfony Blogpost <http://sylius.org/blog/sending-configurable-e-mails-in-symfony>`_.
+
+Currently **Sylius** is sending e-mails only in a few "must-have" cases - see :doc:`E-mails documentation </book/emails>`.
+Of course these cases may not be sufficient for your business needs, if so you will need to create your own, custom e-mails inside your system.
```

Comment:
think this should be `your own, custom, emails` or I prefer `your own custom emails` (no comma).